BrainPortal/app/models/portal_task.rb

```diff
@@ -605,20 +616,49 @@ def empty? #:nodoc:
       @real_errors.empty?
     end
 
+    def full_message(*args) #:nodoc:
+      @real_errors.full_message(*args)
+    end
+
     def full_messages(*args) #:nodoc:
       @real_errors.full_messages(*args)
     end
 
-    def generate_message(paramspath,*args) #:nodoc:
-      @real_errors.generate_message(paramspath.to_la_id,*args)
+    def get(paramspath) #:nodoc:
+      @real_errors.get(paramspath.to_la_id.parameterize.underscore.to_sym)
+    end
+
+    def has_key?(paramspath) #:nodoc:
+      @real_errors.has_key?(paramspath.to_la_id.parameterize.underscore.to_sym)
+    end
+
+    def include?(paramspath) #:nodoc:
+      @real_errors.include?(paramspath.to_la_id.parameterize.underscore.to_sym)
+    end
+
+    def keys #:nodoc:
+      @real_errors.keys
+    }
```

Comment:
I meant to say there's a spurious closing bracket at line 641 that makes the entire file not even load, because of the syntax error...

```
Bianca params_errors models > ruby -c portal_task.rb 
portal_task.rb:641: syntax error, unexpected tSTRING_DEND, expecting keyword_end
portal_task.rb:707: syntax error, unexpected tSTRING_DEND, expecting '}'
  def self.human_attribute_name(attname,options={})
                                                  ^
portal_task.rb:709: syntax error, unexpected tSTRING_DEND, expecting '}'
portal_task.rb:736: syntax error, unexpected tSTRING_DEND, expecting '}'
...ibutes(old_params, options = {}) #:nodoc:
...                               ^
portal_task.rb:737: syntax error, unexpected tSTRING_DEND, expecting '}'
portal_task.rb:739: warning: circular argument reference - options
portal_task.rb:739: syntax error, unexpected tSTRING_DEND, expecting '}'
portal_task.rb:818: syntax error, unexpected keyword_end, expecting end-of-input
```